sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -248,7 +284,6 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
       return
     }
     checkSurfaceSizeChanged()
-    drainQueue(renderEventQueue)
```

Comment:
is it intended? is render event queue even drained at all now?
in any case will be revisiting it in https://github.com/mapbox/mapbox-maps-android/issues/1044

Reply:
Returned